Update locale & twemproxy

On ppc64le, upstream `make ci-build` resulted in the following error(s) : 

```
Step 18/63 : RUN sudo runuser -l postgres -c     "${POSTGRES_PREFIX}/bin/initdb --pgdata='${POSTGRES_DATA_PREFIX}/data' --auth='trust'" < /dev/null     && sudo runuser -l postgres -c     "${POSTGRES_PREFIX}/bin/postgres -D '${POSTGRES_DATA_PREFIX}/data'"       > /tmp/postgres.log 2>&1 & sleep 5     && sudo runuser -l postgres -c "${POSTGRES_PREFIX}/bin/createdb test"     && sudo runuser -l postgres -c "${POSTGRES_PREFIX}/bin/psql test"
 ---> Running in 9acba8a3c1ee
The files belonging to this database system will be owned by user "postgres".
This user must also own the server process.

initdb: invalid locale settings; check LANG and LC_* environment variables
createdb: could not connect to database template1: could not connect to server: No such file or directory
	Is the server running locally and accepting
	connections on Unix domain socket "/tmp/.s.PGSQL.5432"?
```

& twemproxy version `v0.4.1` cannot be built, as it does not recognize underlying architecture.
And is fixed in `v0.5.0` 

```
config/config.guess: unable to guess system type

This script, last modified 2009-04-27, has failed to recognize
the operating system you are using. It is advised that you
download the most up to date version of the config scripts from

  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.guess;hb=HEAD
and
  http://git.savannah.gnu.org/gitweb/?p=config.git;a=blob_plain;f=config.sub;hb=HEAD

If the version you run (config/config.guess) is already up to date, please
send the following data and any information you think might be
pertinent to <config-patches@gnu.org> in order to provide the needed
information to handle your system.

config.guess timestamp = 2009-04-27

uname -m = ppc64le
uname -r = 4.18.0-240.10.1.el8_3.ppc64le
uname -s = Linux
uname -v = #1 SMP Mon Jan 18 17:21:08 UTC 2021

/usr/bin/uname -p = ppc64le
/bin/uname -X     = 

hostinfo               = 
/bin/universe          = 
/usr/bin/arch -k       = 
/bin/arch              = ppc64le
/usr/bin/oslevel       = 
/usr/convex/getsysinfo = 

UNAME_MACHINE = ppc64le
UNAME_RELEASE = 4.18.0-240.10.1.el8_3.ppc64le
UNAME_SYSTEM  = Linux
UNAME_VERSION = #1 SMP Mon Jan 18 17:21:08 UTC 2021
configure: error: cannot guess build type; you must specify one
configure: error: ./configure failed for contrib/yaml-0.1.4
```

--- a/Dockerfile.ci
+++ b/Dockerfile.ci
@@ -1,13 +1,14 @@
 FROM centos:8
 MAINTAINER Alejandro Martinez Ruiz <amr@redhat.com>
 
-ARG CIRCLECI_TOOLS="git openssh-clients tar gzip ca-certificates"
+ARG CIRCLECI_TOOLS="git openssh-clients tar gzip ca-certificates glibc-locale-source glibc-langpack-en"
 # set localtime to UTC and install basic build tools
 RUN (ln -s /usr/share/zoneinfo/UTC /etc/localtime 2> /dev/null || true) \
     && dnf update -y \
     && dnf install -y sudo git make gcc autoconf automake libtool gettext file \
        ${CIRCLECI_TOOLS} \
-    && dnf clean all
+    && dnf clean all \
+    && localedef -c -f UTF-8 -i en_US en_US.UTF-8
 
 ARG USER_NAME=ruby
 RUN useradd -m -d "/home/${USER_NAME}" -l -G wheel ${USER_NAME} \
@@ -54,9 +55,9 @@ RUN sudo runuser -l postgres -c \
     && sudo runuser -l postgres -c "${POSTGRES_PREFIX}/bin/psql test"
 
 # Twemproxy installation
-ARG TWEMPROXY_VERSION=0.4.1
+ARG TWEMPROXY_VERSION=0.5.0
 RUN curl -sSLf \
-    "https://codeload.github.com/twitter/twemproxy/tar.gz/v${TWEMPROXY_VERSION}" \
+    "https://github.com/twitter/twemproxy/releases/download/0.5.0/twemproxy-${TWEMPROXY_VERSION}.tar.gz" \
     | tar -C "/home/${USER_NAME}" -xz
 
 ARG TWEMPROXY_CONFIGURE_OPTIONS

